source/features/quick-repo-deletion.tsx

```diff
@@ -104,7 +108,7 @@ async function start(buttonContainer: HTMLDetailsElement): Promise<void> {
 async function init(): Promise<void | false> {
 	if (
 		// Only if the user can delete the repository
-		!await elementReady('[data-tab-item="settings-tab"]') ||
+		!await elementReady('nav [data-content="Settings"]') ||
```

Comment:
If this attribute is no longer available, other features also broke. It’s worth checking them out (in a separate PR)